docs/stream-aggregation.md

```diff
@@ -334,35 +336,128 @@ For example, the following config removes the `:1m_sum_samples` suffix added [to
+
+The results of `sum_samples` with aggregation interval of `1m` is equal to the `sum_over_time(some_counter[1m])` query.
+
+For example, see below time series produced by config with aggregation interval `1m` and the regular query:
+
+<img alt="sum_samples aggregation" src="stream-aggregation-check-sum-samples.png">
+
+### last
+
+`last` returns the last input [sample value](https://docs.victoriametrics.com/keyConcepts.html#raw-samples).
+
+The results of `last` with aggregation interval of `1m` is equal to the `last_over_time(some_counter[1m])` query.
+
+This aggregation output doesn't make much sense with `by` lists specified in the [config](#stream-aggregation-config). 
+The result of aggregation by labels in this case will be undetermined, because it depends on the order of processing the time series.
+
+### min
+
+`min` returns the minimum input [sample value](https://docs.victoriametrics.com/keyConcepts.html#raw-samples).
+
+The results of `min` with aggregation interval of `1m` is equal to the `min_over_time(some_counter[1m])` query.
+
+For example, see below time series produced by config with aggregation interval `1m` and the regular query:
+
+<img alt="min aggregation" src="stream-aggregation-check-min.png">
+
+### max
+
+`max` returns the maximum input [sample value](https://docs.victoriametrics.com/keyConcepts.html#raw-samples).
+
+The results of `max` with aggregation interval of `1m` is equal to the `max_over_time(some_counter[1m])` query.
+
+For example, see below time series produced by config with aggregation interval `1m` and the regular query:
+
+<img alt="total aggregation" src="stream-aggregation-check-max.png">
+
+### avg
+
+`avg` returns the average input [sample value](https://docs.victoriametrics.com/keyConcepts.html#raw-samples).
+
+The results of `avg` with aggregation interval of `1m` is equal to the `avg_over_time(some_counter[1m])` query.
+
+For example, see below time series produced by config with aggregation interval `1m` and `by: ["instance"]` and  the regular query:
+
+<img alt="avg aggregation" src="stream-aggregation-check-avg.png">
+
+### stddev
+
+`stddev` returns [standard deviation](https://en.wikipedia.org/wiki/Standard_deviation) for the input [sample values](https://docs.victoriametrics.com/keyConcepts.html#raw-samples).
+
+The results of `stddev` with aggregation interval of `1m` is equal to the `stddev_over_time(some_counter[1m])` query.
+
+### stdvar
+
+`stdvar` returns [standard variance](https://en.wikipedia.org/wiki/Variance) for the input [sample values](https://docs.victoriametrics.com/keyConcepts.html#raw-samples).
+
+The results of `stdvar` with aggregation interval of `1m` is equal to the `stdvar_over_time(some_counter[1m])` query.
+
+For example, see below time series produced by config with aggregation interval `1m` and the regular query:
+
+<img alt="stdvar aggregation" src="stream-aggregation-check-stdvar.png">
```

Comment:
hm, the image shows the discrepancy between rollup and raw query. I'm not sure I understand why it is so...